lib/utils/getFiatUrls/index.ts

```diff
@@ -0,0 +1,11 @@
+import { FIAT } from "../../constants";
+
+export const getFiatRetirementUrl = (params: { isProduction: boolean }) =>
+  params.isProduction
+    ? FIAT.retirementCheckout.production
+    : FIAT.retirementCheckout.staging;
+
+export const getFiatBalanceUrl = (params: { isProduction: boolean }) =>
```

Comment:
we will remove the credit card from KlimaDAO in the near future. So then this wouldn't need to live in `lib` it could actually just go in carbonmark/lib/constants `urls` object like our other ones. this is more clean/consistent IMO.

We can put everything in carbonmark now, even if that results in code duplication in this PR, or we can document it in a ticket and clean it up later

Reply:
Moved to Carbonmark. Deleted on klimadao/lib